### PR TITLE
Ansible plugin versions 2.2.0 and 2.3.0

### PR DESCRIPTION
**Pull request by sbaudoin**

### CODE REVIEW

File: ansible.properties [hunk 1]
```diff
@@ -1,12 +1,24 @@
...
-publicVersions=2.1.0
+archivedVersions=1.0.0,2.0.0,2.2.0
+publicVersions=2.3.0
  
 defaults.mavenGroupId=com.github.sbaudoin
 defaults.mavenArtifactId=ansible
 
+2.3.0.description=Support for SonarQube 8
+2.3.0.date=2019-12-23
+2.3.0.sqVersions=[6.7,LATEST]
```
Comment (ganncamp) [inline ansible.properties]: Only one version can be compatible with `LATEST`

File: ansible.properties [hunk 2]
```diff
@@ -1,12 +1,24 @@
...
+2.3.0.changelogUrl=https://github.com/sbaudoin/sonar-ansible/releases/tag/v2.3.0
+2.3.0.downloadUrl=https://github.com/sbaudoin/sonar-ansible/releases/download/v2.3.0/sonar-ansible-plugin-2.3.0.jar
+
+2.2.0.description=Ability to use and supply custom ansible-lint configuration files
+2.2.0.date=2019-05-30
+2.2.0.sqVersions=[6.7,7.9]
+2.2.0.changelogUrl=https://github.com/sbaudoin/sonar-ansible/releases/tag/v2.2.0
+2.2.0.downloadUrl=https://github.com/sbaudoin/sonar-ansible/releases/download/v2.2.0/sonar-ansible-plugin-2.2.0.jar
```
Comment (ganncamp) [inline ansible.properties]: There is no point in adding this directly into the archived section. Either make it public and compatible with a non-overlapping subset of SQ versions or remove it.

Comment (sbaudoin) [reply ansible.properties]: Version 2.2 was actually missing in this file (I forgot to create the PR for this version) so I added it back for consistency reasons

File: ansible.properties (hunk 1 above)
Comment (ganncamp) [reply ansible.properties]: Could you bump the start of this range up to 7.9, please? Sorry I didn't notice that the first time around.

**Comment by sbaudoin** [comment]
Done

File: ansible.properties [hunk 3]
```diff
@@ -1,15 +1,27 @@
...
-publicVersions=2.1.0
+archivedVersions=1.0.0,2.0.0,2.1.0,2.2.0
+publicVersions=2.3.0
  
 defaults.mavenGroupId=com.github.sbaudoin
 defaults.mavenArtifactId=ansible
 
+2.3.0.description=Support for SonarQube 8
+2.3.0.date=2019-12-23
+2.3.0.sqVersions=[7.9,LATEST]
```
Comment (ganncamp) [inline ansible.properties]: This description says "Support for SQ 8". Should compatibility start there?

Comment (sbaudoin) [reply ansible.properties]: 7 and 8 are both supported with this version. I meant here that this is the first version that supports SQ 8 but not exclusively.

**Pull request merged**